nucypher/network/nodes.py

```diff
@@ -16,49 +16,53 @@
 """
 
 import contextlib
-import datetime
 import time
 from collections import defaultdict, deque
 from contextlib import suppress
 from queue import Queue
-from typing import Iterable, List
-from typing import Set, Tuple, Union
+from typing import Iterable, List, Set, Tuple, Union
 
 import maya
 import requests
+from bytestring_splitter import BytestringSplitter, BytestringSplittingError, PartiallyKwargifiedBytes, \
+    VariableLengthBytestring
+from constant_sorrow import constant_or_bytes
+from constant_sorrow.constants import (CERTIFICATE_NOT_SAVED, FLEET_STATES_MATCH, NOT_SIGNED, NO_KNOWN_NODES,
+                                       NO_STORAGE_AVAILIBLE, RELAX, UNKNOWN_VERSION)
```

Comment:
Small style nitpick (will be good to perform import optimization, sorting, and formatting as part of pre-commit hooks):

```suggestion
from bytestring_splitter import (
    BytestringSplitter,
    BytestringSplittingError,
    PartiallyKwargifiedBytes,
    VariableLengthBytestring
)
from constant_sorrow import constant_or_bytes
from constant_sorrow.constants import (
    CERTIFICATE_NOT_SAVED, 
    FLEET_STATES_MATCH,
    NOT_SIGNED,
    NO_KNOWN_NODES,
    NO_STORAGE_AVAILIBLE,
    RELAX,
    UNKNOWN_VERSION
)
```

Reply:
NO_STORAGE_AVAILIBLE -> NO_STORAGE_AVAILABLE